docs/networks/mainnet/running-a-node.md

```diff
@@ -31,6 +31,18 @@ The system requirements will depend on the clients you are running. Please refer
 | RAM                 | 16 GB          |
 | SSD (NVMe)          | 100 GB         |
 
+## Supported clients versions
+
+The LUKSO network currently supports the following clients versions
```

Comment:
lgtm, but I'd suggest mentioning that running higher versions may cause issues for validator and it's attestations/sync committee participation. So that's why we are supporting up to these releases.

Reply:
We can also say that `:warning: Please be advised that running the newer version improperly may result in triggering a network fork on your node (Dencun), potentially leading to validation issues, loss of funds, or even a validator slash.`